src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -1115,7 +1115,11 @@ public void setMaxRxFrameHeightPx(int maxRxFrameHeightPx)
      */
     public void setSelectedEndpointIds(Set<String> selectedEndpointIds)
     {
-        this.selectedEndpointIds = new HashSet<>(selectedEndpointIds);
+        if (!this.selectedEndpointIds.equals(selectedEndpointIds))
+        {
+            this.selectedEndpointIds = new HashSet<>(selectedEndpointIds);
+            update(lastEndpointOrdering, getAvailableBandwidth());
```

Comment:
Nice catch! Was that causing the failures to switch?

Reply:
i think so, yeah. there may be more calls missing, i'm going through all this now